ethpm/utils/contract.py

```diff
@@ -58,3 +64,14 @@ def generate_contract_factory_kwargs(
     if "runtime_bytecode" in contract_data:
         runtime_bytecode = to_bytes(text=contract_data["runtime_bytecode"]["bytecode"])
         yield "bytecode_runtime", encode_hex(runtime_bytecode)
+
+
+def compile_contracts(contract_name: str, alias: str, paths: List[str]) -> str:
+    '''
+    Compile multiple contracts to bytecode.
+    '''
+    bin_id = '{0}.sol:{0}'.format(contract_name)
+    contract_paths = [PACKAGES_DIR + alias + path[1:] for path in paths]
```

Comment:
Instead of `+` which is a little opaque when building strings, I think using string formatting will be easier to read.

```python
[
    "{dir}{alias}{path}".format(dir=PACKAGES_DIR, alias=alias, path=path[1:])
    for path in paths
]
```